india_stocks_api/internal/fivepaisa/database/master_contract_db.py

```
@@ -0,0 +1,227 @@
+                    # Fallback
+                    instrument_type = 'EQ' if exch_type == 'C' else 'FUT'
+
+            # Parse Dates
+            expiry = None
+            if hasattr(row, 'Expiry') and pd.notna(row.Expiry):
+                try:
+                    # Generic parser or specific? Usually 5paisa has specific format
+                    # Original code used pd.to_datetime logic.
+                    ts = pd.to_datetime(row.Expiry)
+                    expiry = ts.date()
+                except:
```

Ruff (0.14.14)

[error] Do not use bare `except`

(E722)